Bucket_sort/Bucket_sort.c.c

```diff
@@ -0,0 +1,97 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+typedef struct bucket{
+		double value;					// structure for maintaining buckets
+		struct bucket *next;
+}b;
+
+b* create_node(double p){
+	b *x;
+	x=(b *)malloc(sizeof(b));		//create a new node for inserting new input into the bucket
+	x->value=p;
```

Comment:
Fix indentation. You have used various indentation throughout the file. Use 4 space indentation.

Reply:
Okay

Reply:
It is still pending and please fix file name as well.

Reply:
I have added indentation and changed file name

Reply:
There is still multiple indentation across file.

Reply:
I have done changes. Sorry for the inconvenience. I was unable to identify